loky/process_executor.py

```diff
@@ -1053,17 +1031,6 @@ def _start_executor_manager_thread(self):
         if self._executor_manager_thread is None:
             mp.util.debug('_start_executor_manager_thread called')
 
-            # When the executor gets garbarge collected, the weakref callback
-            # will wake up the queue management thread so that it can terminate
-            # if there is no pending work item.
-            def weakref_cb(
-                    _, thread_wakeup=self._executor_manager_thread_wakeup,
-                    shutdown_lock=self._shutdown_lock):
-                mp.util.debug('Executor collected: triggering callback for'
-                              ' QueueManager wakeup')
-                with self._shutdown_lock:
-                    thread_wakeup.wakeup()
-
```

Comment:
This was never used? I wonder if this is not revealing a bug. Maybe @tomMoral remembers the original intent of this code.

Reply:
There is something similar in `_ExecutorManagerThread.__init__` so it looks like this was a left-over of a previous refactoring...